content/3_Silver/Func_Graphs.mdx

```diff
@@ -715,73 +715,98 @@ print(" ".join(str(i + 1) for i in ans))
 
 </PySection>
 </LanguageSection>
-
 ### Solution 3
-
-It's also possible to use `floyd(x)` to generate answers for all vertices in the connected component
-containing `x` without using adjacency lists.
-
+Without using any adjacency lists. `floyd(x)` generates answers for all connected vertices. `fillup(a,b)` fills up the vertices, from `a` to `b`, `b` being the coordinate, whose `ans[b]` has already been calculated `floyd(x)`
```

Comment:
ayo why'd you delete all my stuff

Reply:
Updated your changes. Sorry, had overlooked your commit, above.

Reply:
Is there some precommit hook, that checks, for the length of the text on each row ? It seems to be updating wrt to that. Checked it multiple times with multiple commits, in this pr. Attaching screenshot.

![image](https://github.com/cpinitiative/usaco-guide/assets/8929771/3780cee9-5dfe-43c7-ad26-f1781483cc5c)